Core/HistoManager.h

```diff
@@ -61,7 +61,7 @@ class HistoManager
   
   void Book(); //! call once; book (create) all trees and histograms
   void Save(); //! call once; save all trees and histograms
-  void SaveEvtPack() { fTree->Fill(); };
+  void SaveEvtPack() { /*if (!fEmptyEvent)*/ fTree->Fill(); fEmptyEvent = true; };
```

Comment:
Can the commented part be removed?

Reply:
Yes, sorry, I forgot to remove all test parts